acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/signing/SecpSigningAcceptanceTest.java

```diff
@@ -68,18 +68,15 @@ public void signDataWithKeyInAzure(@TempDir Path tomlDirectory) {
   }
 
   @Test
-  public void signDataWithFileBasedKey(@TempDir Path tomlDirectory)
-      throws IOException, URISyntaxException {
+  public void signDataWithFileBasedKey(@TempDir Path keyConfigDirectory) throws URISyntaxException {
     final String keyPath =
         new File(Resources.getResource("secp256k1/wallet.json").toURI()).getAbsolutePath();
 
-    final Path passwordPath = tomlDirectory.resolve("password");
-    Files.write(passwordPath, "pass".getBytes(UTF_8));
+    // final Path passwordPath = keyConfigDirectory.resolve("password");
```

Comment:
nit: might remove comments if they are not required anymore.

Reply:
done